### PR TITLE
Switched steps 'Install OADP operator' > 'Create OpenShift secret'

### PR DESCRIPTION
**Pull request by HagayVider1**
GH# : 
OCPBUGS# : [OADP-1983](https://issues.redhat.com//browse/OADP-1983)
OSDOCS# : [OADP-1983](https://issues.redhat.com//browse/OADP-1983)

Switched steps 1 and 2 under 'Installing OADP on Red Hat OpenShift Service on AWS with AWS STS' 
1. Install OADP operator...
2. Create an OpenShift secret... 

Version(s): OADP 1.2.0

Issue: https://issues.redhat.com/browse/OADP-1983

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by HagayVider1** [comment]
@kalexand-rh @yuokada126 

OADP-1983 : https://issues.redhat.com/browse/OADP-1983

See switched steps 1 and 2.
Please review and comment.

Thanks

File: modules/oadp-installing-oadp-rosa-sts.adoc [hunk 1]
```diff
@@ -20,6 +20,11 @@ Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the
 
 .Procedure
 
+
+. Install the OADP Operator.
+.. In the {product-title} web console, navigate to Operators *->* OperatorHub.
```
Comment (mburke5678) [inline modules/oadp-installing-oadp-rosa-sts.adoc:25]: @HagayVider1 I see this is existing text. But, can you s/Operators *->* OperatorHub/*Operators* -> *OperatorHub*

**Comment by mburke5678** [comment]
@HagayVider1 One nit to fix an existing format error. Can you post a direct preview of the change? Also, is [OADP-1983](https://issues.redhat.com//browse/OADP-1983) the correct issue for this change? That issue seems to be asking for an explanation of `role_arn`.

**Comment by amastbau** [comment]
@mburke5678 
https://github.com/openshift/openshift-docs/pull/62093/files
I think this is the one you are looking for.
but the preview link is missing.

**Pull request closed without merging**